### PR TITLE
Alt/command key mapping options

### PR DESCRIPTION
**Pull request by ulrikdamm**
I've really wanted to expose the LeftOptionIsAlt/RightOptionIsAlt and LeftCommandIsCtrl/RightCommandIsCtrl wine config options, to be able to use alt+key to write special characters on the international keyboard and use command+c/v for copy/paste. I've exposed those options in Wine.swift, and added them as options in the general settings UI.

I also changed the UI slightly to fit everything, and put some labels on the different sections. I wasn't to use what to call the option-button options, so I've gone with "Windows alt" or "macOS option" for now.

The one thing missing is adding these options to the NativeAOT config, because right now, they're reset every time you relaunch, but I haven't yet delved into that. I would have to look into how to even clone and compile it, which I can do eventually if needed. Feel free to reject the pull request if you want that supported first, or if this is an unwanted feature in general.

### CODE REVIEW

File: XIV on Mac/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -154,9 +155,20 @@ Do not enable this on non-retina displays, unless for screenshot purposes.";
 "SETTINGS_PLATFORM_STEAM" = "Steam";
 "SETTINGS_FREE_TRIAL" = "Free Trial";
 "SETTINGS_GENERAL_FREE_TRIAL_BLURB" = "When using the Free Trial, you must choose the same license type you used when you signed up for the trial. Additionally, your Final Fantasy XIV account must already have an active trial, as XIV on Mac cannot sign you up automatically.";
+"SETTINGS_GENERAL_TITLE_NETWORK" = "Network";
 "SETTINGS_DOWNLOAD_LIMIT" = "Limit Download speed to";
 "SETTINGS_DOWNLOAD_LIMIT_PLACEHOLDER" = "0.0";
 "SETTINGS_DOWNLOAD_LIMIT_UNITS" = "MB/sec";
+"SETTINGS_DOWNLOAD_LIMIT_UNITS" = "MB/sec";
```
Comment (marzent) [inline XIV on Mac/en.lproj/Localizable.strings:162]: These line seems to be double now; on a side note can you copy paste the newly added strings to the other languages as well until they are localized?

**Comment by seathasky** [comment]
This is a good idea but with your current situation (international keyboard), does software like "Karabiner Elements" not able to solve this niche issue for you?

**Review by marzent — CHANGES_REQUESTED** [review]

**Review by ulrikdamm — COMMENTED** [review]
Fixed the indentation, sorry about that. Also removed the accidental duplicate strings entry, and added the new strings to the other languages.

**Comment by seathasky** [comment]
> As for using 3rd party software to do key remapping, it seems like a complicated way to solve the problem? Especially when wine already has configuration built-in for that exact purpose. But if the problem seems to niche to dedicate UI space for it, maybe then just a change to have it not reset those keys in the registry? I have changed it in the regedit before, but it resets with each relaunch.

Not arguing against it. I like your idea actually and think it's a great addition. It's just not something we normally have brought up for the client. I was actually mostly curious if that method worked for you or if you have tried it personally.

**Comment by ulrikdamm** [comment]
I can vouch for it working, when I change from windows alt to macOS alt I can type special characters, like alt+a for å, which I normally can't :) the downside of course is that the game won't recognise the alt button for things like hotbar keybinds, but that's why having the options makes sense.

**Review by marzent — APPROVED** [review]

**Comment by ulrikdamm** [comment]
I'm not sure how exactly it gets reset. I saw that the retina scale option was referenced in the Settings.syncToXL method, so I assumed that these options would have to be synced the same way, but now I see that it just passes the value. I'll try to look into why it happens.

**Pull request merged**